src/Interfaces/interface.ts

```diff
@@ -0,0 +1,38 @@
+import { Dispatch } from 'react';
+import { TodoAction } from './actions';
+
+export type TodoItem = {
+  id: number;
+  content: string;
+  done: boolean;
+};
```

Comment:
```suggestion
export interface TodoItem {
  id: number;
  content: string;
  done: boolean;
};
```

interface가 조금 더 적합해 보입니다 ㅎㅎ

Reply:
저에게는 타입스크립트의 인터페이스가 자바의 `interface`같은 느낌이 너무 강하다보니,,, 아무래도 이런건 `type`에 더 가깝지 않나? 라고 생각하게 만드는게 있나봅니다..